---
content_type: page
title: Related Resources
uid: 06bfd03e-4047-a2b6-8e72-0625cb9b8a57
---

Literature Searches

When trying to identify papers from the primary literature, you may find it useful to use the National Center for Biotechnology Information ([NCBI](http://www.ncbi.nlm.nih.gov/)) website. This site hosts [PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed), an online literature search resource. Many of the titles and abstracts that can be reached through PubMed also have links to online versions (PDFs or full text versions) of the associated papers.

Another online literature search tool is the [Web of Science database](http://www.isiwebofknowledge.com/index.html). This tool is similar in some respects to PubMed, except that it is not so strongly focused on medically-related journals, and includes journals not indexed on PubMed.

Web of Science also enables searches of the Science Citation Index, which allows you to ask the question, "What papers have been written recently that cite a paper I already have?" This type of search is very useful for keeping up-to-date in a research field.

  
DNA Sequence Analysis

Sequence searches can be carried out using the GenBank search tools available on the NCBI site. Most valuable of these is probably the [Entrez](http://www.ncbi.nlm.nih.gov/) browser, which can be used to find DNA sequences based upon keyword searches.

If you have a DNA sequence and would like to ask whether other sequences are known with homology to your sequence, use the [BLAST tool (BLASTn)](http://www.ncbi.nlm.nih.gov/blast/Blast.cgi?CMD=Web&LAYOUT=TwoWindows&AUTO_FORMAT=Semiauto&ALIGNMENTS=50&ALIGNMENT_VIEW=Pairwise&CLIENT=web&DATABASE=nr&DESCRIPTIONS=100&ENTREZ_QUERY=(none)&EXPECT=10&FILTER=L&FORMAT_OBJECT=Alignment&FORMAT_TYPE=HTML&HIT). With this tool, you can paste your sequence into a window and the NCBI server will compare it to all known sequences, pulling up anything that matches.

On the other hand, if you have 2 sequences and want to know whether these two specifically match, you can use the "[BLAST 2 Sequences tool](https://www.ncbi.nlm.nih.gov/pubmed/10339815)".

In some cases, rather than searching for DNA sequences that closely match your own sequence, you may want to identify proteins whose sequences are homologous to those encoded by your DNA element. To do this, use the [BLASTx tool](http://www.ncbi.nlm.nih.gov/blast/Blast.cgi?CMD=Web&LAYOUT=TwoWindows&AUTO_FORMAT=Semiauto&ALIGNMENTS=50&ALIGNMENT_VIEW=Pairwise&CLIENT=web&DATABASE=nr&DESCRIPTIONS=100&ENTREZ_QUERY=(none)&EXPECT=10&FILTER=L&FORMAT_OBJECT=Alignment&FORMAT_TYPE=HTML&GENETIC_CODE=1&HITLIST_SIZE=100&NCBI_GI=on&PAGE=Translations&PROGRAM=blastx&SERVICE=plain&SET_DEFAULTS.x=37&SET_DEFAULTS.y=5&SHOW_OVERVIEW=on&UNGAPPED_ALIGNMENT=no&END_OF_HTTPGET=Yes&SHOW_LINKOUT=yes). This tool will translate the DNA sequence you enter and will compare it to known protein sequences or translation products of known DNA sequences.

While trying to understand how different biochemical pathways fit together in a cell, or when you are simply trying to see a snapshot of an entire physiological pathway, it is often easier to forgo the usual perusal of Stryer and to instead find a useful website. [The Kyoto Encyclopedia of Genes and Genomes](https://www.genome.jp/kegg/) is very useful for giving you such a snapshot of metabolism. The site also provides links to enzymes, structures and other useful information.

If you are specifically interested in pathways involved in the degradation of xenobiotics (nasty environmental toxins like naphthalene, toluene, benzene, PCBs, etc.) you may want to check out [The University of Minnesota Biocatalysis/Biodegradation Database.](https://academic.oup.com/nar/article/38/suppl_1/D488/3112190)